packages/dropdown-component/package.json

```diff
@@ -3,10 +3,11 @@
   "version": "1.5.0",
   "description": "Stencil dropdown component",
   "main": "dist/index.cjs.js",
-  "module": "dist/custom-elements/index.js",
-  "es2015": "dist/esm/index.mjs",
-  "es2017": "dist/esm/index.mjs",
-  "types": "dist/custom-elements/index.d.ts",
+  "module": "dist/index.js",
```

Comment:
I am not sure if this is something we should do. The version 2.0 from stencil explicitly change this check release notes at https://github.com/ionic-team/stencil/releases/tag/v2.0.0

Reply:
yeah we are using that, you can compare the values